### DISCUSSION TITLE
Team Builder - Open to all Ideas!

### DISCUSSION THREAD

**Discussion by TeamBuilderAPI** [TeamBuilderAPI/TeamBuilder#1] [Ideas]
Discussing Team Builder and it's potential, all ideas are welcome!

**Comment by TeamBuilderAPI** [comment]
Idea - Work together with all other Team Builder apps as some form of roster share, that means that there is no competition for this tool specifically, just need a standard for all and any Team Builder platforms to talk to each other. This does not mean that data is coming from the same app, it means that all apps contribute to a global roster using their own data which would be read in by this app idea as a common roster language.

**Reply by TeamBuilderApp** [reply]
We moved here: https://github.com/TeamBuilderApp/TeamBuilder

**Comment by TeamBuilderAPI** [comment]
Idea - Team Builder would eventually build the worlds first non-professional team to challenge and win against a real pro-team.

**Reply by TeamBuilderApp** [reply]
We moved here: https://github.com/TeamBuilderApp/TeamBuilder

**Comment by TeamBuilderAPI** [comment]
Idea is to build a common Roster language so that ALL Team Builder apps can work together with anyone world-wide! The largest possible global roster is the goal!

**Reply by TeamBuilderApp** [reply]
We moved here: https://github.com/TeamBuilderApp/TeamBuilder

**Comment by TeamBuilderApp** [comment]
We moved here: https://github.com/TeamBuilderApp/TeamBuilder
